Create an issue template for tracking issues (#533)
NixOS/nix.dev: .github/ISSUE_TEMPLATE/tracking-issue.md

---
name: Tracking Issue

> about: Create a tracking issue for a project or piece of work

Reviewer: ```suggestion
about: Create a tracking issue for a collection of issues and pull-requests
```

> title: Tracking Issue - PROJECT DESCRIPTION

Reviewer: ```suggestion
```

labels: tracking

> assignees: ''
> 

Reviewer: ```suggestion
```

---

## Project description
<!-- Briefly describe the project to provide context for the work that needs to be done -->

## Steps

> <!-- List the steps that are required to deliver the project, linking issues and PRs for each step as they are opened -->

Reviewer: ```suggestion
<!-- List the steps that are required to finish the project, and link issues and pull requests -->
```
